package.json

```
@@ -22,7 +22,8 @@
 		"proto-gen:object": "buf generate packages/object/src/proto -o packages/object/src/proto",
 		"proto-gen:network": "buf generate packages/network/src/proto -o packages/network/src/proto",
 		"release": "release-it",
-		"test": "vitest"
+		"test": "vitest",
+		"build-examples": "pnpm --filter 'ts-drp-example*' build"
```

```suggestion
		"build": "pnpm build:packages && pnpm build:examples",
		"build:packages": "pnpm --filter '@ts-drp/*' build",
		"build:examples": "pnpm --filter 'ts-drp-example*' build"
```

- add the build scripts for everything
- change the postinstall to run `pnpm build:packages`
- put it in alphabetical order

(refactor the `.replit` with this changes)

---

@sfroment u missed the comment after the suggestion

---

@d-roak shall be good ! 
for the 1st point that was the suggestions right ?